### PR TITLE
Add workflow images

### LINKED ISSUE #107: Add "workflow" graphic to landing page
https://www.figma.com/file/0ifTqGA2I3Y216eCJxh1rr/Keyman-Predictive-Text?node-id=1%3A2

### PR DESCRIPTION
**Pull request by jienius**
Resolves #107

### CODE REVIEW

**Review by xqiu1 — APPROVED** [review]
LGTM

File: src/app/pages/LandingPage.svelte [hunk 1]
```diff
@@ -31,15 +31,16 @@
...
+    --wave-height: 222px;
+    --masthead-space-top: 1rem;
+    --masthead-content-height: 13rem;
     --masthead-solid-color-height: calc(
       var(--masthead-space-top) + var(--masthead-content-height)
     );
     --masthead-background-color: #09f;
 
     width: 100vw;
+    max-width: 100%;
```
Comment (eddieantonio) [inline src/app/pages/LandingPage.svelte:43]: Why is this needed?

File: src/app/pages/LandingPage.svelte [hunk 2]
```diff
@@ -59,6 +60,7 @@
     display: flex;
     width: 80vw;
     max-width: var(--body-max-width);
+    min-width: 600px;
```
Comment (eddieantonio) [inline src/app/pages/LandingPage.svelte]: Ah, this helps prevent "tearing" of the background and the wave! Could you please [change it to `rem`](https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#css-prefer-rem-to-px-for-positioning):

```suggestion
    min-width: 37.5rem;
```

File: src/app/pages/LandingPage.svelte [hunk 3]
```diff
@@ -186,10 +212,36 @@
...
+          alt=""
+          role="presentation" />
+      </div>
+
+      <img
+        src="assets/workflow-keyboard.svg"
+        alt=""
+        role="presentation"
+        class="explanation__workflow__image" />
+    </div>
```
Comment (eddieantonio) [inline src/app/pages/LandingPage.svelte]: Since this "explanation" is _content_ (it's briefly explaining what the app does) and not just decorative, I'd add `alt=` attributes and put everything in a [`<figure>` element]():

```suggestion
    <figure class="explanation__workflow">
      <img
        src="assets/workflow-spreadsheet.svg"
        alt="a spreadsheet of words in your language"
        class="explanation__workflow__image" />

      <div class="explanation__workflow__transform">
        Predictive Text Studio
        <img
          class="explanation__workflow__transform__arrow"
          src="assets/workflow-arrow.svg"
          alt=""
          role="presentation" />
      </div>

      <img
        src="assets/workflow-keyboard.svg"
        alt="your words appear in the predictive text banner on your phone's keyboard"
        class="explanation__workflow__image" />
    </figure>
```

File: src/app/pages/LandingPage.svelte (hunk 1 above)
Comment (jienius) [reply src/app/pages/LandingPage.svelte:43]: Because there is now a vertical scroll bar, the body size is no longer 100vw, setting the width to 100vw will result in a horizontal scroll bar
